Makefile

```diff
@@ -1,3 +1,12 @@
+# Check the current Sail version
+SAIL_VERSION := $(shell etc/version_check.sh > /dev/null; echo $$?)
```

Comment:
Now that the new Sail version has been released - and CI for this repo automatically uses the latest version - we should be able to do something like

```
SAIL_VERSION := $(shell $(SAIL) --version --require-version 0.18 > /dev/null; echo $$?)
```

Also I'd suggest calling it `SAIL_VERSION_STATUS` or `SAIL_VERSION_ERROR` or something because it isn't actually the version.

Alternatively...

```
SAIL += --require-version 0.18
```

Reply:
It seems like just adding `--require-version 0.18` to `SAIL_FLAGS` is probably the simplest and cleanest solution. And it gives a fairly descriptive error message on its own: `Sail version 0.17.1 is older than requested version 0.18`.

Reply:
I made a new PR for that here: #534